plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts

```diff
@@ -47,31 +54,49 @@ export function addHistoricalEventsExportCapability(
         // the historical export to duplicate them
         meta.global.timestampBoundariesForTeam = timestampBoundaries
 
-        await meta.utils.cursor.init('batch_id')
+        await meta.utils.cursor.init(BATCH_ID_CURSOR_KEY)
 
         const storedTimestampCursor = await meta.storage.get(TIMESTAMP_CURSOR_KEY, null)
-        if (storedTimestampCursor) {
-            await meta.jobs.restartExportIfNeeded({ storedTimestampCursor }).runIn(10, 'minutes')
-        }
+        await meta.storage.set(OLD_TIMESTAMP_CURSOR_KEY, storedTimestampCursor || 0)
+        await meta.storage.set(RUN_EVERY_MINUTE_LAST_RUN_KEY, Date.now() + TEN_MINUTES)
 
         await oldSetupPlugin?.()
     }
 
-    tasks.job['restartExportIfNeeded'] = {
-        name: 'restartExportIfNeeded',
-        type: PluginTaskType.Job,
-        exec: async (payload) => {
-            const storedTimestampCursor = await meta.storage.get(TIMESTAMP_CURSOR_KEY, null)
+    tasks.schedule.runEveryMinute = {
+        name: 'runEveryMinute',
+        type: PluginTaskType.Schedule,
+        exec: async () => {
+            const lastRun = await meta.storage.get(RUN_EVERY_MINUTE_LAST_RUN_KEY, 0)
+            const exportShouldBeRunning = await meta.storage.get(EXPORT_RUNNING_KEY, false)
+
+            const have10MinutesPassed = Date.now() - Number(lastRun) < TEN_MINUTES
```

Comment:
nit: same here try to not refer to the value, but rather what the value measures, so if we change it to 15min it's easier with code changes (both `have10MinutesPassed` & `TEN_MINUTES`

Reply:
I mean, I get the sentiment. But these are constants. If we change the value, we can change the name